pkgs/development/python-modules/pyparser/default.nix

```diff
@@ -0,0 +1,56 @@
+{ buildPythonPackage, fetchPypi, fetchpatch, python, stdenv, parse }:
+
+let
+  # pyparser only builds with parse == 1.6.5
+  parse = buildPythonPackage rec {
```

Comment:
As mentioned by @FRidh, this should not be. I don't know how to do this, though.

Reply:
Btw I also don't understand why github marks the other comments on this diff as outdated, I did not change this part of the diff in my fixup commits!